Lib/python/pyrun.swg

```diff
@@ -1422,8 +1467,10 @@ SWIG_Python_TypeQuery(const char *type)
     descriptor = SWIG_TypeQueryModule(swig_module, swig_module, type);
     if (descriptor) {
       obj = PyCapsule_New((void*) descriptor, NULL, NULL);
-      PyDict_SetItem(cache, key, obj);
-      Py_DECREF(obj);
+      if (obj) {
```

Comment:
Further OOM fix: `PyCapsule_New` may return `NULL`, in which case `PyDict_SetItem` should not be called with a `NULL` value (it will be dereferenced & crash - or trigger `assert(value)` in the debug build).